### DISCUSSION TITLE
有谁遇到过error:0607A082这种问题吗

### DISCUSSION THREAD

**Discussion by sunkangchina** [w7corp/easywechat#2603] [Q&A]
用5.x可以正常收到通知
用6.x提示
error:0607A082:digital envelope routines:EVP_CIPHER_CTX_set_key_length:invalid key

**Comment by TheNorthMemory** [comment]
这是openssl抛的异常，最末尾 `invalid key`  说明你给的通知验签(apiv2密钥 及或 apiv3密钥) 不对所致。

**Reply by sunkangchina** [reply]
感谢
